custom_components/tech/tech.py

```diff
@@ -37,7 +37,11 @@ def __init__(
 
         """
         _LOGGER.debug("Init Tech")
-        self.headers = {"Accept": "application/json", "Accept-Encoding": "gzip"}
+        self.headers = {
+            "Accept": "application/json",
+            "Accept-Encoding": "gzip",
+            "User-Agent": "TechController/1.0 (Home Assistant Integration)"
```

Comment:
it is hard to tell how tech company will react on that, this will give them chance to cut off this integration very easy by blocking this header